### PR TITLE
Improved test coverage of piral-cli injectors

### PR DESCRIPTION
**Pull request by int0x81**
# New Pull Request

For more information, see the `CONTRIBUTING` guide.

## Prerequisites

Please make sure you can check the following boxes:

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project
- [x] All new and existing tests passed

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [ ] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have updated the documentation accordingly
- [x] I have added tests to cover my changes

### Description

This pull request improves the test coverage of the piral-cli injectors
- Bumped test coverage of _/src/tooling/piral-cli/src/injectors/piral.ts_ to **96,5%**
- Bumped test coverage of _/src/tooling/piral-cli/src/injectors/pilet.ts_ to **69,12%**

### Remarks

Two unit tests caused stackoverflows at first due to infinite recursion in the following methods:
- handle() in _/src/tooling/piral-cli/src/injectors/pilet.ts_ 
- sendResponse() in _/src/tooling/piral-cli/src/injectors/pilet.ts_

I placed termination conditions for the resursive flow in both of these methods

### CODE REVIEW

File: src/tooling/piral-cli/src/injectors/piral.ts [hunk 1]
```diff
@@ -32,11 +32,16 @@ export default class PiralInjector implements KrasInjector {
 
   setOptions() {}
 
-  sendResponse(path: string, target: string, dir: string, url: string): KrasResponse {
+  sendResponse(path: string, target: string, dir: string, url: string, recursionDepth: number = 0): KrasResponse {
```
Comment (FlorianRappl) [inline src/tooling/piral-cli/src/injectors/piral.ts:35]: Just `recursionDepth = 0` (the `number` is inferred).

File: src/tooling/piral-cli/src/injectors/piral.ts [hunk 2]
```diff
@@ -32,11 +32,16 @@ export default class PiralInjector implements KrasInjector {
...
+
+    if(recursionDepth > 10) {
```
Comment (FlorianRappl) [inline src/tooling/piral-cli/src/injectors/piral.ts:37]: Please use prettier for your formatting (VS Code command palette "Format Document") -> space between `if` and the `(`.

Comment (FlorianRappl) [reply src/tooling/piral-cli/src/injectors/piral.ts:37]: Maybe make the 10 a const on the top of the file. Also I think ~4 should be sufficient (or even 3).

File: src/tooling/piral-cli/src/injectors/piral.ts (hunk 1 above)
Comment (int0x81) [reply src/tooling/piral-cli/src/injectors/piral.ts]: adapted; removed the type declaration

File: src/tooling/piral-cli/src/injectors/piral.ts (hunk 2 above)
Comment (int0x81) [reply src/tooling/piral-cli/src/injectors/piral.ts]: adapted; ran prettier, and created the constant `maxRetrySendResponse` with value of 4

**Pull request merged**